--- a/src/slave/kprop_util.c	2017-09-25 11:28:22.000000000 -0500
+++ b/src/slave/kprop_util.c	2018-01-14 17:24:09.271047875 -0600
@@ -32,6 +32,10 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 
+#ifndef MAXHOSTNAMELEN
+#define MAXHOSTNAMELEN 256
+#endif
+
 /*
  * Convert an IPv4 or IPv6 socket address to a newly allocated krb5_address.
  * There is similar code elsewhere in the tree, so this should possibly become
