--- a/lib/util/utimens.c	2016-12-18 07:52:06.000000000 -0600
+++ b/lib/util/utimens.c	2017-11-24 15:52:44.724962310 -0600
@@ -54,6 +54,14 @@
 # define MTIME_TO_TIMEVAL(_x, _y)      do { (_x)->tv_sec = (_y)->st_mtime; (_x)->tv_usec = 0; } while (0)
 #endif /* HAVE_ST_MTIM */
 
+/* Time flags for futimens. */
+#define UTIME_NOW   -1  /* corresponds to the current time */
+#define UTIME_OMIT  -2  /* target time is omitted */ 
+
+#define AT_FDCWD		-100    /* Special value used to indicate
+                                           openat should use the current
+                                           working directory. */
+
 /*
  * Convert the pair of timespec structs passed to futimens() / utimensat()
  * to a pair of timeval structs, handling UTIME_OMIT and UTIME_NOW.
